javascript-data-model/order-history.js

```diff
@@ -5,11 +5,15 @@ var orderHistory = [
     shipTo: 'JS Masher',
     orderNumber: '114-3941689-8772232',
     delDate: '08/08/20',
-    imgUrl: ['https://www.example.com/img1'],
-    itemName: ['Javascript for impatient programmers'],
-    author: ['Rauschmayer, Dr. Axel'],
-    returnWindow: '09/07/20',
-    cost: [31.55],
+    itemsBought: [
+      {
+        imgUrl: 'https://www.example.com/img1',
+        itemName: 'Javascript for impatient programmers',
+        author: 'Rauschmayer, Dr. Axel',
+        cost: 31.55,
+        returnWindow: '09/07/20'
+      }
+    ],
     items: 1
```

Comment:
You can remove this property, that data is already available from the length of the `itemsBought` array